core/src/main/scala/org/scalawag/jibe/backend/ubuntu/UpdateAptGet.sh

```diff
@@ -0,0 +1,13 @@
+PATH=/bin:/usr/sbin:/usr/bin
+
+lastUpdatedTime=$(stat -c %Y /var/cache/apt/pkgcache.bin)
+currentTime="$(date +'%s')"
+currentCacheAge=$(( currentTime - lastUpdatedTime ))
+
+echo "Current cache age is $currentCacheAge"
+echo "Maximum acceptable cache age (in seconds) is $refreshInterval"
+
+if [ "${currentCacheAge}" -gt "$refreshInterval" ]; then
+    echo "Running apt-get update"
+    apt-get update &
```

Comment:
Tests for this fail unless I run it in the apt-get in the background, but it seems wrong and forces me to add a sleep after this test (I'll comment on that later).